gunnik/client/webpack.config.js

```diff
@@ -1,16 +1,21 @@
 const path = require('path');
+const CopyPlugin = require("copy-webpack-plugin");
 
 module.exports = {
-    entry: './src/Index.tsx',
+    entry: {
+        './artefact': './src/Index.tsx',
+        './background/background': './src/background/background.ts',
```

Comment:
Nice! Tak aspoň že to buildí ze src. 
,ale ... můj packager mi to zabalil podle manifestu, tím pádem kdybych měl více souborů nebo je přejmenoval tak se mi to nerozbije.

Ale za mě tohle stačí.

Reply:
Tohle beru jako uzavřené